parser_library/test/lsp/lsp_context_instr_test.cpp

```diff
@@ -93,3 +93,25 @@ TEST_F(lsp_context_instr, ADDFRR_loaded_changed_instr_set)
     EXPECT_TRUE(result_z15);
     EXPECT_TRUE(result_xa);
 }
+
+TEST(lsp_completion_instr, consistency)
```

Comment:
Is it intended to have the test in this file as it is related solely to instructions? `lsp_completion_test.cpp` might be an alternative location for the test

Reply:
I was considering the other file as well, but that one contains tests that focus on transforming the internal representation of the response to user-visible text. So I decided to put it next the tests that are related to instruction completions.